_source/integrations/jaeger.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: article
+title: Store Jaeger traces in Logz.io
+permalink: /integrations/store-jaeger-traces.html
+description: Here's how you can use Logz.io as data storage for Jaeger traces.
+open-source:
+  - title: Jaeger-Logz.io Trace Storage
+    github-repo: jaeger-logzio
+contributors:
+  - imnotashrimp
+  - yyyogev
+---
+
+Jaeger-Logz.io is a storage option for Jaeger.
+It allows Jaeger to store distributed traces on your Logz.io account.
+
+This integration requires Logz.io API access.
+The Logz.io API is available for Pro and Enterprise accounts.
+{:.info-box.note}
+
+### Limitations
+
+When you use the Jaeger UI to find traces stored in Logz.io,
+there are a couple limitations.
+For most users, these won't be an issue,
+but they're still good to know:
+
+* **Lookback** must be 48 hours or less
+* **Limit Results** must be 1000 traces or less
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Deploy a single container](#single-container-config)
+* [Deploy separate containers](#separate-containers-config)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="single-container-config">
+
+These instructions cover deployment
+of all the necessary pieces
+(Jaeger agent, collector, and query service)
+in a single, all-in-one Docker container.
+
+#### Deploy everything in a single container
+
+<div class="tasklist">
+
+##### Create a Docker network
+
+```shell
+docker network create net-logzio
+```
+
+##### Run the container
+
+You can configure the Logz.io extension with shell variables or environment variables.
+
+For a complete list of options, see the parameters below the code block. 👇
+
+```shell
+docker run -d \
+  -e ACCOUNT_TOKEN=<<SHIPPING-TOKEN>> \
+  -e API_TOKEN=<<API-TOKEN>> \
+  --name=jaeger-logzio \
+  --network=net-logzio \
+  -p 5775:5775/udp \
+  -p 6831:6831/udp \
+  -p 6832:6832/udp \
+  -p 5778:5778 \
+  -p 16686:16686 \
+  -p 14268:14268 \
+  -p 9411:9411 \
+logzio/jaeger-logzio:latest
+```
+
+###### Environment variables
+
+| Parameter | Description |
+|---|---|
+| ACCOUNT_TOKEN <span class="required-param"></span> | Required when using as a collector to ship traces to Logz.io. <br> {% include log-shipping/replace-vars.html token=true %} |
+| API_TOKEN	<span class="required-param"></span> | Required to read back traces from Logz.io. <br> Replace `<<API-TOKEN>>` with an [API token](https://app.logz.io/#/dashboard/settings/api-tokens) from the account you want to use. |
+| REGION <span class="default-param">`us`</span> | Two-letter region code. This determnies your listener URL (where you're shipping the logs to) and API URL. <br> Replace `us` with your region's code. |
```

Comment:
Shouldn't we add a link to the account region?

Reply:
Added.